packages/ckeditor5-utils/src/madewith.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/madewith
+ */
+
+/**
+ * Possible states of the key after verification
+ */
+export type VerifiedKeyStatus = 'VALID' | 'INVALID' | 'EXPIRED';
```

Comment:
Why do we need EXPIRED? Do we distinguish such a state in the premium features license check mechanism?

Reply:
I think that when we will have such kind of information we can prepare special url for, for example, expired state - to track it and/or to redirect to some specific landing page or something similar. @mlewand WDYT?

Reply:
Available suffixes for premium features: `Valid`, `Invalid`, `Trial:*`